controllers/mirrorpeer_controller.go

```diff
@@ -161,9 +186,64 @@ func processMirrorPeerSecretChanges(ctx context.Context, rc client.Client, mirro
 	return anyErr
 }
 
+func (r *MirrorPeerReconciler) checkTokenExchangeStatus(ctx context.Context, mp multiclusterv1alpha1.MirrorPeer) (bool, error) {
+	logger := log.FromContext(ctx)
+	pr1 := mp.Spec.Items[0]
+	pr2 := mp.Spec.Items[1]
```

Comment:
add a todo for when we support more than 2.

Reply:
done